utils/frame/remote-externalities/Cargo.toml

```diff
@@ -29,6 +29,8 @@ sp-core = { version = "4.0.0-dev", path = "../../../primitives/core" }
 sp-runtime = { version = "4.0.0-dev", path = "../../../primitives/runtime" }
 sp-version = { version = "4.0.0-dev", path = "../../../primitives/version" }
 
+frame-support = { version = "4.0.0-dev", path = "../../../frame/support" }
```

Comment:
Why?

Reply:
The compilation kept failing till I added this dependency.

Reply:
Strange. Wondering why this is. Seems unrelated to this PR. Maybe you need to merge master?

Reply:
Yeah, it is. I did merge master before sending this PR. Will do so again.

Reply:
I merged master and removed the dependency and the test failed. Should I leave it then?

Reply:
![image](https://user-images.githubusercontent.com/23155127/140496385-23a12b55-17ec-4292-93c8-42f4ae5e1dda.png)

This is what shows.

Reply:
Yeah. My feeling is that something in the CI image was maybe changed. Every pending PR needs to add this line to not fail. Leave it in please move it to `dev-dependencies` as its only used in tests.

Like here: https://github.com/paritytech/substrate/pull/9993/files#diff-fe6fa7aabc85c3fadcb402dc4b2e00618e68ebf6371e80ce3dbfdb4ee9f0b22e

Reply:
Alright, will do so.